Lib/hashlib.py

```diff
@@ -235,15 +235,15 @@ def prf(msg, inner=inner, outer=outer):
         loop = 1
         from_bytes = int.from_bytes
         while len(dkey) < dklen:
-            prev = prf(salt + loop.to_bytes(4, 'big'))
+            prev = prf(salt + loop.to_bytes(4))  # big endian
```

Comment:
The comment is not needed here. Se the one at the next line.
```suggestion
            prev = prf(salt + loop.to_bytes(4))
```